in-core_joplin_case_study.ipynb

```diff
@@ -66,9 +66,21 @@
     "  - Retrieve existing metadata for various hazards\n",
     "  - Search hazards per specific information\n",
     "  - Request creation of a new hazard\n",
-    "  - Request deletion of a new hazard\n",
-    "\n",
-    "For more information on **pyincore** modules, see the documentation here: https://incore.ncsa.illinois.edu/doc/pyincore/index.html\n"
+    "  - Request deletion of a hazard\n",
+    "- **DFR3Service:** similar to the **HazardService**, once the `IncoreClient` has been initiated, the `DFR3Service` module uses the \n",
+    "  information from that client to administer existing information about known or new fragility, restoration, repair and recovery curves.     `DFR3Service` can:\n",
+    "  - Retrieve existing curves, mappings and metadata \n",
+    "  - Search fragility curves and mappings per specific information\n",
+    "  - Request creation of new curves and mappings\n",
+    "  - Request deletion of a curves and mappings\n",
+    "- **DataService:** similar to the **HazardService**, once the `IncoreClient` has been initiated, the `DataService` module uses the \n",
+    "  information from that client to administer existing information about known or new datasets. `DataService` can:\n",
+    "  - Retrieve existing metadata for various datasets\n",
```

Comment:
Data Service can also download the data. Would it make sense to add one more bullet point?

Reply:
Done, I added it to the first bullet